### PR TITLE
Audio block: add alignment and editing

### PR DESCRIPTION
**Pull request by notnownikki**

### CODE REVIEW

File: blocks/library/audio/index.js [hunk 1]
```diff
@@ -29,15 +31,51 @@ registerBlockType( 'core/audio', {
...
 		},
+		align: {
+			type: 'string',
+		},
 	},
 
-	edit( { attributes, setAttributes, className } ) {
-		const { src } = attributes;
+	getEditWrapperProps( attributes ) {
+		const { align, width } = attributes;
```
Comment (timmyc) [inline blocks/library/audio/index.js]: I don't think we need the `width` attribute here, or in the returned object below.

**Review by timmyc — COMMENTED** [review]
One minor comment on un-needed attribute, otherwise this is looking great and testing out perfectly.  Thanks for fixing this up @notnownikki

File: blocks/library/audio/index.js (hunk 1 above)
Comment (notnownikki) [inline blocks/library/audio/index.js]: Absolutely correct, symptom of taking this from the image block. Fixing now!

**Review by timmyc — APPROVED** [review]
LGTM 🚢

**Pull request merged**